site/content/docs/authentication/multiple-headers.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Auth: Multiple Header Based Authentication"
+type: page
+layout: decision-action
+action: Configure ZAP to use your auth tokens
+---
+
+You can do this via:
+* [Replacer Rules](/docs/getting-further/authentication/handling-auth-yourself/#replacer-rules)
+* [Scripts](/docs/getting-further/authentication/handling-auth-yourself/#scripts)
+
+##### Background
+
+You have more than one authentication tokens which you need to supply in HTTP headers.
```

Comment:
make tokens singular

Reply:
Had to read that twice to realise it was wrong ;) Done..